backend/director/tools/videodb_tool.py

```
@@ -51,21 +66,26 @@ def create_collection(self, name, description=""):
         try:
             self.collection.delete()
-            return {"success": True, "message": f"Collection {self.collection.id} deleted successfully"}
+            return {
+                "success": True,
+                "message": f"Collection {self.collection.id} deleted successfully",
+            }
         except Exception as e:
-            raise Exception(f"Failed to delete collection {self.collection.id}: {str(e)}")
+            raise Exception(
+                f"Failed to delete collection {self.collection.id}: {str(e)}"
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)